www-nginx/ngx-metrics/ngx-metrics-0.1.1.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MY_PN="ngx_metrics"
+NGINX_MOD_S="${WORKDIR}/${MY_PN}-${PV}"
+
+inherit nginx-module
+
+DESCRIPTION="An NGINX module that exports the server's metrics"
+HOMEPAGE="https://github.com/liquidm/ngx_metrics"
+SRC_URI="
+	https://github.com/liquidm/ngx_metrics/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz
+"
+
+LICENSE="BSD-2"
```

Comment:
Until we know what it is, it is `all-rights-reserved` and should be `RESTRICT="mirror bindist"` (not that I plan on merging it right now anyway, just saying in general).